apps/emqx_bridge_http/src/emqx_bridge_http_schema.erl

```diff
@@ -39,48 +47,124 @@ fields("put") ->
     fields("config");
 fields("get") ->
     emqx_bridge_schema:status_fields() ++ fields("post");
-fields("creation_opts") ->
+%%--- v1 bridges config file
+%% see: emqx_bridge_schema:fields(bridges)
+fields("config") ->
+    basic_config() ++ request_config();
+%%--------------------------------------------------------------------
+%% v2: configuration
+fields(action) ->
+    %% XXX: Do we need to rename it to `http`?
+    {webhook,
+        mk(
+            hoconsc:map(name, ref(?MODULE, webhook_action)),
+            #{
+                desc => <<"HTTP Action Config">>,
+                required => false
+            }
+        )};
+fields(webhook_action) ->
+    [
+        {enable, mk(boolean(), #{desc => ?DESC("config_enable"), default => true})},
+        {connector,
+            mk(binary(), #{
+                desc => ?DESC(emqx_connector_schema, "connector_field"), required => true
+            })},
+        {description, emqx_schema:description_schema()},
+        %% Note: there's an implicit convention in `emqx_bridge' that,
+        %% for egress bridges with this config, the published messages
+        %% will be forwarded to such bridges.
+        {local_topic,
```

Comment:
if this was not presented to users from UI in v1 design.
we can make this a hidden config in v2.